semantic-python/test/Test.hs

```diff
@@ -69,15 +72,20 @@ assertJQExpressionSucceeds directive tree core = do
   catch @_ @Streaming.Process.ProcessExitedUnsuccessfully jqPipeline $ \err -> do
     HUnit.assertFailure (unlines [errorMsg, dirMsg, jsonMsg, astMsg, treeMsg, treeMsg', show err])
 
-fixtureTestTreeForFile :: HasCallStack => FilePath -> Tasty.TestTree
-fixtureTestTreeForFile fp = HUnit.testCaseSteps fp $ \step -> withFrozenCallStack $ do
-  fileContents <- ByteString.readFile ("semantic-python/test/fixtures" </> fp)
+fixtureTestTreeForFile :: HasCallStack => Path.RelFile -> Tasty.TestTree
+fixtureTestTreeForFile fp = HUnit.testCaseSteps (Path.toString fp) $ \step -> withFrozenCallStack $ do
+  let fullPath = Path.relDir "semantic-python/test/fixtures" </> fp
```

Comment:
In contrast to my above comments re: `Path.RelFile` in core annotations, I think this is a superb example of the value proposition of `Path.RelFile`. I’m definitely in favour, and think we should use it in `semantic` and `semantic-ast` and the like (cc @aymannadeem) as well 👍